MarsRoverTest/MarsRoverTest.UnitTest/RoverTests/WhenMovingForward.cs

```diff
@@ -0,0 +1,58 @@
+﻿using MarsRover;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MarsRoverTest.UnitTest.RoverTests
+{
+    [TestClass]
+    public class WhenMovingForward
+    {
+        [TestMethod]
+        public void AndFacingNorthThenYIncrementsByOne()
+        {
+            //arrange 
+            var rover = new Rover();
```

Comment:
Total nitpick, but we can combine these two statements to a single line as 
`var rover = new Rover{Direction=Direction.North}`

Reply:
Hey Cameron this change has been done for each direction in order to make things look cleaner thanks for the heads up on the cleaner syntax!